lib/napa/setup.rb

```diff
@@ -18,7 +18,7 @@ def skip_initialization=(value)
     end
 
     def env
-      @_env ||= ActiveSupport::StringInquirer.new(ENV['RACK_ENV'] || 'development')
+      @_env ||= ActiveSupport::StringInquirer.new(ENV['RACK_ENV'] || ENV['RAILS_ENV'] || 'development')
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.